packages/pagination/src/mongo-pagination-param.decorator.ts

```diff
@@ -35,18 +35,20 @@ interface MongoPaginationOptions {
   pageName?: string;
   perPageName?: string;
   defaultLimit?: number;
+  exclude?: string[];
```

Comment:
How about naming this `excludedMongoOperators`? Because `exclude` alone is not really precise on what we are excluding.

Reply:
Good idea! 😉  but how about `excludedFields` to be more generic? Because it can specify any fields to exclude from the request query, not only Mongo operators.

Reply:
or `excludedKeys`?

Reply:
Oh right, I did not think about this. Then `excludedKeys` seems appropriate for me.